utilities/transactions/pessimistic_transaction_db.h

```diff
@@ -223,13 +229,16 @@ class WritePreparedTxnDB : public PessimisticTransactionDB {
         heap_.pop();
         erased_heap_.pop();
       }
+      while (heap_.empty() && !erased_heap_.empty()) {
+        erased_heap_.pop();
```

Comment:
TIL std::priority_queue::clear() doesn't exists.

Reply:
Yeah it does not. Not sure what you wanted to imply though.

Reply:
Just feeling surprised about the std interface.